src/main/java/com/solr/clientwrapper/infrastructure/adaptor/OrderPojoSolrAdapter.java

```diff
@@ -0,0 +1,40 @@
+package com.solr.clientwrapper.infrastructure.adaptor;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.solr.clientwrapper.domain.port.spi.SolrOrderPersistencePort;
+import com.solr.clientwrapper.infrastructure.entity.SolrOrderPojo;
+import com.solr.clientwrapper.infrastructure.repository.SolrOrderRepository;
+
+@Service
+@Transactional
+public class OrderPojoSolrAdapter implements SolrOrderPersistencePort {
```

Comment:
Rename the class as its name doesnt clearly explain what it does

Reply:
Done.